---
title: "Laurits Fredsgaard Larsen"
<!--date: 2022-08-13T22:37:49+02:00-->
draft: false
---


A passionate and results-oriented **Machine Learning & Data Science** professional with a solid foundation in **Molecular Biomedicine**. I excel in **bioinformatics**, **research**, and software development, and am proficient in **Python**, **R**, and **C++** programming languages. My skills include analyzing **complex data** and implementing cutting-edge machine learning algorithms for a wide range of applications in the **life sciences** field.

## Education
### 2020â€“2023 BSc Machine Learning & Data Science, University of Copenhagen
GPA: 11.3

- ğŸ’¡ Completed masters level courses on **machine learning theory**, **advanced deep learning theory**, and **generative graphical models**, where I gained a deep understanding of the mathematical foundations of machine learning, **advanced architectures** for deep learning, and the ability to model **complex distributions** and **data generating processes** using graphical models.
- ğŸ“ Attended the **Gaussian Process and Uncertainty Quantification Summer School** at The University of Sheffield, UK (Sept 2022) where I gained hands-on experience with **Gaussian Processes** and their applications in machine learning and **optimization**.

### 2018â€“2020 MSc Molecular Biomedicine, University of Copenhagen
GPA: 10.8

- ğŸ”¬ Conducted a **head-to-head study** comparing two different conjugation systems and virus-backbones for antigen display, investigating the effect on the antibody response in mice.
- ğŸ“ Published the findings in the MDPI journal Vaccines (2021)

### 2015â€“2018 BSc Molecular Biomedicine, University of Copenhagen
GPA: 9.0

## Work Experience
### May 2021â€“present: Bioinformatics Student Assistant, LEO Pharma A/S, Ballerup
- ğŸ–¥ï¸ Developed a **backend** utilizing **Python** and **Django** for handling **high-dimensional data**, providing **API access** for integration with multiple front-end interfaces.
- ğŸ¤ Worked in a **shared codebase (git)** for efficient collaboration.
- ğŸ” Designed and implemented **data analysis pipelines** for **gene expression data**.

### Aprâ€“Jul 2020: Research Assistant, Center for Medical Parasitology, University of Copenhagen
- ğŸ¦  Assisted in the development of **next-generation vaccine** against SARS-CoV-2 by creating spike-protein variants in insect cells.
- ğŸ“– Contributed to the publication "Capsid-like particles decorated with the SARS-CoV-2 receptor-binding domain elicit strong virus neutralization activity" in the journal **Nature Communications** (2021).

## Skills
- **Programming:** Python, R, C++
- **Data Science:** Data visualization, data wrangling, data analysis
- **Machine Learning:** Supervised learning, deep learning, generative models
- **Biology:** Molecular biology, biochemistry, immunology
- **Languages:** Danish (Native), English (Fluent), Japanese (Business level, JLPT N2)

## Volunteering
### 2019â€“present: Network Administrator, Industrikollegiet, Copenhagen
- ğŸŒ Self-taught **networking skills** and administrated the **network infrastructure** for the dormitory, troubleshoot and resolved network issues, trained and assisted other residents.

### 2022â€“2023: Treasurer, Industrikollegiet, Copenhagen
- ğŸ’° Managed **budget** and **finances** for the resident council, assisted in the planning and facilitation of events and activities for the dormitory community.

## Projects
### 2022-2023: A Fully Convolutional Neural Network Model for Masked Amino Acid Position Identification
- ğŸ§ª Using a **3D U-Net architecture** to learn the amino acid propensities from voxel representations of protein structures.
- ğŸ” Investigating the effects of masking different portions of the protein structure on the accuracy of the model.

### 2022: Generative Latent Variable Model for Protein Sequences
- ğŸ§¬ Trained a **variational autoencoder** using multiple sequence alignment as input.
- ğŸ“ˆ Compared the performance of the model to a similar model using importance weighted autoencoder.

### 2021: Developed a Stand-Mixer Tracker System
- ğŸ“ Utilized **machine learning** and web technologies to predict and display the location of a shared stand-mixer in real-time.
- ğŸŒ Implemented a **random forest classifier** and a dashboard using **Plotly Dash**.
- ğŸ› ï¸ Used an **ESP8266 microcontroller** to gather location data, and **PostgreSQL**, **Flask**, and **Docker** for data storage and deployment.

## Scholarships
- 2018â€“2023: Peter og Emma Thomsens Legat
  - Monthly stipend awarded based on outstanding academic performance.
- 2018, 2019: Novo Nordisk International Talent Program
  - Summer schools at National University of Singapore and University of Tokyo.
- 2017: Scandinavia-Japan Sasakawa Foundation, Rudolph Als Fondet, IngeniÃ¸r Ad. ClÃ©ments Legat
  - University exchange at Osaka University.

## Publications
1. *Fougeroux C, GoksÃ¸yr L, \[...\], **Fredsgaard L**, \[...\]*: Capsid-like particles decorated with the SARS-CoV-2 receptor-binding domain elicit strong virus neutralization activity. *Nature Communications*. 2021 [**link**](https://doi.org/10.1038/s41467-020-20251-8)
2. ***Fredsgaard L**, \[...\]*: Head-to-Head Comparison of Modular Vaccines Developed Using Different Capsid Virus-Like Particle Backbones and Antigen Conjugation System. *MDPI Vaccines*. 2021 [**link**](https://doi.org/10.3390/vaccines9060539)

