<!doctype html><html lang=en dir=auto><head><meta charset=utf-8><meta http-equiv=x-ua-compatible content="IE=edge"><meta name=viewport content="width=device-width,initial-scale=1,shrink-to-fit=no"><meta name=robots content="index, follow"><title>Laurits Fredsgaard Larsen | Laurits Fredsgaard Larsen</title><meta name=keywords content><meta name=description content="A passionate and results-oriented Machine Learning & Data Science student with a solid foundation in Molecular Biomedicine. I excel in bioinformatics, research, and software development, and am proficient in Python, R, and C++ programming languages. My skills include analyzing complex data and implementing cutting-edge machine learning algorithms for a wide range of applications in the life sciences field.
Education 2020â€“2023 BSc Machine Learning & Data Science, University of Copenhagen GPA: 11."><meta name=author content><link rel=canonical href=https://laurits.me/cv/><link crossorigin=anonymous href=/assets/css/stylesheet.bc1149f4a72aa4858d3a9f71462f75e5884ffe8073ea9d6d5761d5663d651e20.css integrity="sha256-vBFJ9KcqpIWNOp9xRi915YhP/oBz6p1tV2HVZj1lHiA=" rel="preload stylesheet" as=style><script defer crossorigin=anonymous src=/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG+9vmJ0cTS+ovo0FeA=" onload=hljs.initHighlightingOnLoad()></script>
<link rel=icon href=https://laurits.me/favicon.ico><link rel=icon type=image/png sizes=16x16 href=https://laurits.me/favicon-16x16.png><link rel=icon type=image/png sizes=32x32 href=https://laurits.me/favicon-32x32.png><link rel=apple-touch-icon href=https://laurits.me/apple-touch-icon.png><link rel=mask-icon href=https://laurits.me/safari-pinned-tab.svg><meta name=theme-color content="#2e2e33"><meta name=msapplication-TileColor content="#2e2e33"><noscript><style>#theme-toggle,.top-link{display:none}</style><style>@media(prefers-color-scheme:dark){:root{--theme:rgb(29, 30, 32);--entry:rgb(46, 46, 51);--primary:rgb(218, 218, 219);--secondary:rgb(155, 156, 157);--tertiary:rgb(65, 66, 68);--content:rgb(196, 196, 197);--hljs-bg:rgb(46, 46, 51);--code-bg:rgb(55, 56, 62);--border:rgb(51, 51, 51)}.list{background:var(--theme)}.list:not(.dark)::-webkit-scrollbar-track{background:0 0}.list:not(.dark)::-webkit-scrollbar-thumb{border-color:var(--theme)}}</style></noscript><meta property="og:title" content="Laurits Fredsgaard Larsen"><meta property="og:description" content="A passionate and results-oriented Machine Learning & Data Science student with a solid foundation in Molecular Biomedicine. I excel in bioinformatics, research, and software development, and am proficient in Python, R, and C++ programming languages. My skills include analyzing complex data and implementing cutting-edge machine learning algorithms for a wide range of applications in the life sciences field.
Education 2020â€“2023 BSc Machine Learning & Data Science, University of Copenhagen GPA: 11."><meta property="og:type" content="article"><meta property="og:url" content="https://laurits.me/cv/"><meta property="article:section" content><meta name=twitter:card content="summary"><meta name=twitter:title content="Laurits Fredsgaard Larsen"><meta name=twitter:description content="A passionate and results-oriented Machine Learning & Data Science student with a solid foundation in Molecular Biomedicine. I excel in bioinformatics, research, and software development, and am proficient in Python, R, and C++ programming languages. My skills include analyzing complex data and implementing cutting-edge machine learning algorithms for a wide range of applications in the life sciences field.
Education 2020â€“2023 BSc Machine Learning & Data Science, University of Copenhagen GPA: 11."><script type=application/ld+json>{"@context":"https://schema.org","@type":"BreadcrumbList","itemListElement":[{"@type":"ListItem","position":1,"name":"Laurits Fredsgaard Larsen","item":"https://laurits.me/cv/"}]}</script><script type=application/ld+json>{"@context":"https://schema.org","@type":"BlogPosting","headline":"Laurits Fredsgaard Larsen","name":"Laurits Fredsgaard Larsen","description":"A passionate and results-oriented Machine Learning \u0026amp; Data Science student with a solid foundation in Molecular Biomedicine. I excel in bioinformatics, research, and software development, and am proficient in Python, R, and C++ programming languages. My skills include analyzing complex data and implementing cutting-edge machine learning algorithms for a wide range of applications in the life sciences field.\nEducation 2020â€“2023 BSc Machine Learning \u0026amp; Data Science, University of Copenhagen GPA: 11.","keywords":[],"articleBody":"A passionate and results-oriented Machine Learning \u0026 Data Science student with a solid foundation in Molecular Biomedicine. I excel in bioinformatics, research, and software development, and am proficient in Python, R, and C++ programming languages. My skills include analyzing complex data and implementing cutting-edge machine learning algorithms for a wide range of applications in the life sciences field.\nEducation 2020â€“2023 BSc Machine Learning \u0026 Data Science, University of Copenhagen GPA: 11.3\nğŸ’¡ Completed masters level courses on machine learning theory, advanced deep learning theory, and generative graphical models, where I gained a deep understanding of the mathematical foundations of machine learning, advanced architectures for deep learning, and the ability to model complex distributions and data generating processes using graphical models. ğŸ“ Attended the Gaussian Process and Uncertainty Quantification Summer School at The University of Sheffield, UK (Sept 2022) where I gained hands-on experience with Gaussian Processes and their applications in machine learning and optimization. 2018â€“2020 MSc Molecular Biomedicine, University of Copenhagen GPA: 10.8\nğŸ”¬ Conducted a head-to-head study comparing two different conjugation systems and virus-backbones for antigen display, investigating the effect on the antibody response in mice. ğŸ“ Published the findings in the MDPI journal Vaccines (2021) 2015â€“2018 BSc Molecular Biomedicine, University of Copenhagen GPA: 9.0\nWork Experience May 2021â€“present: Bioinformatics Student Assistant, LEO Pharma A/S, Ballerup ğŸ–¥ï¸ Developed a backend utilizing Python and Django for handling high-dimensional data, providing API access for integration with multiple front-end interfaces. ğŸ¤ Worked in a shared codebase (git) for efficient collaboration. ğŸ” Designed and implemented data analysis pipelines for gene expression data. Aprâ€“Jul 2020: Research Assistant, Center for Medical Parasitology, University of Copenhagen ğŸ¦  Assisted in the development of next-generation vaccine against SARS-CoV-2 by creating spike-protein variants in insect cells. ğŸ“– Contributed to the publication â€œCapsid-like particles decorated with the SARS-CoV-2 receptor-binding domain elicit strong virus neutralization activityâ€ in the journal Nature Communications (2021). Skills Programming: Python, R, C++ Data Science: Data visualization, data wrangling, data analysis Machine Learning: Supervised learning, deep learning, generative models Biology: Molecular biology, biochemistry, immunology Languages: Danish (Native), English (Fluent), Japanese (Business level, JLPT N2) Volunteering 2019â€“present: Network Administrator, Industrikollegiet, Copenhagen ğŸŒ Self-taught networking skills and administrated the network infrastructure for the dormitory, troubleshoot and resolved network issues, trained and assisted other residents. 2022â€“2023: Treasurer, Industrikollegiet, Copenhagen ğŸ’° Managed budget and finances for the resident council, assisted in the planning and facilitation of events and activities for the dormitory community. Projects 2022-2023: A Fully Convolutional Neural Network Model for Masked Amino Acid Position Identification ğŸ§ª Using a 3D U-Net architecture to learn the amino acid propensities from voxel representations of protein structures. ğŸ” Investigating the effects of masking different portions of the protein structure on the accuracy of the model. 2022: Generative Latent Variable Model for Protein Sequences ğŸ§¬ Trained a variational autoencoder using multiple sequence alignment as input. ğŸ“ˆ Compared the performance of the model to a similar model using importance weighted autoencoder. 2021: Developed a Stand-Mixer Tracker System ğŸ“ Utilized machine learning and web technologies to predict and display the location of a shared stand-mixer in real-time. ğŸŒ Implemented a random forest classifier and a dashboard using Plotly Dash. ğŸ› ï¸ Used an ESP8266 microcontroller to gather location data, and PostgreSQL, Flask, and Docker for data storage and deployment. Scholarships 2018â€“2023: Peter og Emma Thomsens Legat Monthly stipend awarded based on outstanding academic performance. 2018, 2019: Novo Nordisk International Talent Program Summer schools at National University of Singapore and University of Tokyo. 2017: Scandinavia-Japan Sasakawa Foundation, Rudolph Als Fondet, IngeniÃ¸r Ad. ClÃ©ments Legat University exchange at Osaka University. Publications Fougeroux C, GoksÃ¸yr L, [â€¦], Fredsgaard L, [â€¦]: Capsid-like particles decorated with the SARS-CoV-2 receptor-binding domain elicit strong virus neutralization activity. Nature Communications. 2021 link Fredsgaard L, [â€¦]: Head-to-Head Comparison of Modular Vaccines Developed Using Different Capsid Virus-Like Particle Backbones and Antigen Conjugation System. MDPI Vaccines. 2021 link ","wordCount":"640","inLanguage":"en","datePublished":"0001-01-01T00:00:00Z","dateModified":"0001-01-01T00:00:00Z","mainEntityOfPage":{"@type":"WebPage","@id":"https://laurits.me/cv/"},"publisher":{"@type":"Organization","name":"Laurits Fredsgaard Larsen","logo":{"@type":"ImageObject","url":"https://laurits.me/favicon.ico"}}}</script></head><body id=top><script>localStorage.getItem("pref-theme")==="dark"?document.body.classList.add("dark"):localStorage.getItem("pref-theme")==="light"?document.body.classList.remove("dark"):window.matchMedia("(prefers-color-scheme: dark)").matches&&document.body.classList.add("dark")</script><header class=header><nav class=nav><div class=logo><a href=https://laurits.me/ accesskey=h title="Laurits Fredsgaard Larsen (Alt + H)">Laurits Fredsgaard Larsen</a><div class=logo-switches><button id=theme-toggle accesskey=t title="(Alt + T)"><svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24" fill="none" stroke="currentcolor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"><path d="M21 12.79A9 9 0 1111.21 3 7 7 0 0021 12.79z"/></svg><svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24" fill="none" stroke="currentcolor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"><circle cx="12" cy="12" r="5"/><line x1="12" y1="1" x2="12" y2="3"/><line x1="12" y1="21" x2="12" y2="23"/><line x1="4.22" y1="4.22" x2="5.64" y2="5.64"/><line x1="18.36" y1="18.36" x2="19.78" y2="19.78"/><line x1="1" y1="12" x2="3" y2="12"/><line x1="21" y1="12" x2="23" y2="12"/><line x1="4.22" y1="19.78" x2="5.64" y2="18.36"/><line x1="18.36" y1="5.64" x2="19.78" y2="4.22"/></svg></button></div></div><ul id=menu><li><a href=https://laurits.me/cv title=CV><span class=active>CV</span></a></li><li><a href=https://lauritz.me title="Looking for Lauritz HilsÃ¸e?"><span>Looking for Lauritz HilsÃ¸e?</span>&nbsp;<svg fill="none" shape-rendering="geometricPrecision" stroke="currentcolor" stroke-linecap="round" stroke-linejoin="round" stroke-width="2.5" viewBox="0 0 24 24" height="12" width="12"><path d="M18 13v6a2 2 0 01-2 2H5a2 2 0 01-2-2V8a2 2 0 012-2h6"/><path d="M15 3h6v6"/><path d="M10 14 21 3"/></svg></a></li></ul></nav></header><main class=main><article class=post-single><header class=post-header><h1 class=post-title>Laurits Fredsgaard Larsen</h1><div class=post-meta></div></header><div class=post-content><p>A passionate and results-oriented <strong>Machine Learning & Data Science</strong> student with a solid foundation in <strong>Molecular Biomedicine</strong>. I excel in <strong>bioinformatics</strong>, <strong>research</strong>, and software development, and am proficient in <strong>Python</strong>, <strong>R</strong>, and <strong>C++</strong> programming languages. My skills include analyzing <strong>complex data</strong> and implementing cutting-edge machine learning algorithms for a wide range of applications in the <strong>life sciences</strong> field.</p><h2 id=education>Education<a hidden class=anchor aria-hidden=true href=#education>#</a></h2><h3 id=20202023-bsc-machine-learning--data-science-university-of-copenhagen>2020â€“2023 BSc Machine Learning & Data Science, University of Copenhagen<a hidden class=anchor aria-hidden=true href=#20202023-bsc-machine-learning--data-science-university-of-copenhagen>#</a></h3><p>GPA: 11.3</p><ul><li>ğŸ’¡ Completed masters level courses on <strong>machine learning theory</strong>, <strong>advanced deep learning theory</strong>, and <strong>generative graphical models</strong>, where I gained a deep understanding of the mathematical foundations of machine learning, <strong>advanced architectures</strong> for deep learning, and the ability to model <strong>complex distributions</strong> and <strong>data generating processes</strong> using graphical models.</li><li>ğŸ“ Attended the <strong>Gaussian Process and Uncertainty Quantification Summer School</strong> at The University of Sheffield, UK (Sept 2022) where I gained hands-on experience with <strong>Gaussian Processes</strong> and their applications in machine learning and <strong>optimization</strong>.</li></ul><h3 id=20182020-msc-molecular-biomedicine-university-of-copenhagen>2018â€“2020 MSc Molecular Biomedicine, University of Copenhagen<a hidden class=anchor aria-hidden=true href=#20182020-msc-molecular-biomedicine-university-of-copenhagen>#</a></h3><p>GPA: 10.8</p><ul><li>ğŸ”¬ Conducted a <strong>head-to-head study</strong> comparing two different conjugation systems and virus-backbones for antigen display, investigating the effect on the antibody response in mice.</li><li>ğŸ“ Published the findings in the MDPI journal Vaccines (2021)</li></ul><h3 id=20152018-bsc-molecular-biomedicine-university-of-copenhagen>2015â€“2018 BSc Molecular Biomedicine, University of Copenhagen<a hidden class=anchor aria-hidden=true href=#20152018-bsc-molecular-biomedicine-university-of-copenhagen>#</a></h3><p>GPA: 9.0</p><h2 id=work-experience>Work Experience<a hidden class=anchor aria-hidden=true href=#work-experience>#</a></h2><h3 id=may-2021present-bioinformatics-student-assistant-leo-pharma-as-ballerup>May 2021â€“present: Bioinformatics Student Assistant, LEO Pharma A/S, Ballerup<a hidden class=anchor aria-hidden=true href=#may-2021present-bioinformatics-student-assistant-leo-pharma-as-ballerup>#</a></h3><ul><li>ğŸ–¥ï¸ Developed a <strong>backend</strong> utilizing <strong>Python</strong> and <strong>Django</strong> for handling <strong>high-dimensional data</strong>, providing <strong>API access</strong> for integration with multiple front-end interfaces.</li><li>ğŸ¤ Worked in a <strong>shared codebase (git)</strong> for efficient collaboration.</li><li>ğŸ” Designed and implemented <strong>data analysis pipelines</strong> for <strong>gene expression data</strong>.</li></ul><h3 id=aprjul-2020-research-assistant-center-for-medical-parasitology-university-of-copenhagen>Aprâ€“Jul 2020: Research Assistant, Center for Medical Parasitology, University of Copenhagen<a hidden class=anchor aria-hidden=true href=#aprjul-2020-research-assistant-center-for-medical-parasitology-university-of-copenhagen>#</a></h3><ul><li>ğŸ¦  Assisted in the development of <strong>next-generation vaccine</strong> against SARS-CoV-2 by creating spike-protein variants in insect cells.</li><li>ğŸ“– Contributed to the publication &ldquo;Capsid-like particles decorated with the SARS-CoV-2 receptor-binding domain elicit strong virus neutralization activity&rdquo; in the journal <strong>Nature Communications</strong> (2021).</li></ul><h2 id=skills>Skills<a hidden class=anchor aria-hidden=true href=#skills>#</a></h2><ul><li><strong>Programming:</strong> Python, R, C++</li><li><strong>Data Science:</strong> Data visualization, data wrangling, data analysis</li><li><strong>Machine Learning:</strong> Supervised learning, deep learning, generative models</li><li><strong>Biology:</strong> Molecular biology, biochemistry, immunology</li><li><strong>Languages:</strong> Danish (Native), English (Fluent), Japanese (Business level, JLPT N2)</li></ul><h2 id=volunteering>Volunteering<a hidden class=anchor aria-hidden=true href=#volunteering>#</a></h2><h3 id=2019present-network-administrator-industrikollegiet-copenhagen>2019â€“present: Network Administrator, Industrikollegiet, Copenhagen<a hidden class=anchor aria-hidden=true href=#2019present-network-administrator-industrikollegiet-copenhagen>#</a></h3><ul><li>ğŸŒ Self-taught <strong>networking skills</strong> and administrated the <strong>network infrastructure</strong> for the dormitory, troubleshoot and resolved network issues, trained and assisted other residents.</li></ul><h3 id=20222023-treasurer-industrikollegiet-copenhagen>2022â€“2023: Treasurer, Industrikollegiet, Copenhagen<a hidden class=anchor aria-hidden=true href=#20222023-treasurer-industrikollegiet-copenhagen>#</a></h3><ul><li>ğŸ’° Managed <strong>budget</strong> and <strong>finances</strong> for the resident council, assisted in the planning and facilitation of events and activities for the dormitory community.</li></ul><h2 id=projects>Projects<a hidden class=anchor aria-hidden=true href=#projects>#</a></h2><h3 id=2022-2023-a-fully-convolutional-neural-network-model-for-masked-amino-acid-position-identification>2022-2023: A Fully Convolutional Neural Network Model for Masked Amino Acid Position Identification<a hidden class=anchor aria-hidden=true href=#2022-2023-a-fully-convolutional-neural-network-model-for-masked-amino-acid-position-identification>#</a></h3><ul><li>ğŸ§ª Using a <strong>3D U-Net architecture</strong> to learn the amino acid propensities from voxel representations of protein structures.</li><li>ğŸ” Investigating the effects of masking different portions of the protein structure on the accuracy of the model.</li></ul><h3 id=2022-generative-latent-variable-model-for-protein-sequences>2022: Generative Latent Variable Model for Protein Sequences<a hidden class=anchor aria-hidden=true href=#2022-generative-latent-variable-model-for-protein-sequences>#</a></h3><ul><li>ğŸ§¬ Trained a <strong>variational autoencoder</strong> using multiple sequence alignment as input.</li><li>ğŸ“ˆ Compared the performance of the model to a similar model using importance weighted autoencoder.</li></ul><h3 id=2021-developed-a-stand-mixer-tracker-system>2021: Developed a Stand-Mixer Tracker System<a hidden class=anchor aria-hidden=true href=#2021-developed-a-stand-mixer-tracker-system>#</a></h3><ul><li>ğŸ“ Utilized <strong>machine learning</strong> and web technologies to predict and display the location of a shared stand-mixer in real-time.</li><li>ğŸŒ Implemented a <strong>random forest classifier</strong> and a dashboard using <strong>Plotly Dash</strong>.</li><li>ğŸ› ï¸ Used an <strong>ESP8266 microcontroller</strong> to gather location data, and <strong>PostgreSQL</strong>, <strong>Flask</strong>, and <strong>Docker</strong> for data storage and deployment.</li></ul><h2 id=scholarships>Scholarships<a hidden class=anchor aria-hidden=true href=#scholarships>#</a></h2><ul><li>2018â€“2023: Peter og Emma Thomsens Legat<ul><li>Monthly stipend awarded based on outstanding academic performance.</li></ul></li><li>2018, 2019: Novo Nordisk International Talent Program<ul><li>Summer schools at National University of Singapore and University of Tokyo.</li></ul></li><li>2017: Scandinavia-Japan Sasakawa Foundation, Rudolph Als Fondet, IngeniÃ¸r Ad. ClÃ©ments Legat<ul><li>University exchange at Osaka University.</li></ul></li></ul><h2 id=publications>Publications<a hidden class=anchor aria-hidden=true href=#publications>#</a></h2><ol><li><em>Fougeroux C, GoksÃ¸yr L, [&mldr;], <strong>Fredsgaard L</strong>, [&mldr;]</em>: Capsid-like particles decorated with the SARS-CoV-2 receptor-binding domain elicit strong virus neutralization activity. <em>Nature Communications</em>. 2021 <a href=https://doi.org/10.1038/s41467-020-20251-8><strong>link</strong></a></li><li><em><strong>Fredsgaard L</strong>, [&mldr;]</em>: Head-to-Head Comparison of Modular Vaccines Developed Using Different Capsid Virus-Like Particle Backbones and Antigen Conjugation System. <em>MDPI Vaccines</em>. 2021 <a href=https://doi.org/10.3390/vaccines9060539><strong>link</strong></a></li></ol></div><footer class=post-footer><ul class=post-tags></ul></footer></article></main><footer class=footer><span>&copy; 2023 <a href=https://laurits.me/>Laurits Fredsgaard Larsen</a></span>
<span>Powered by
<a href=https://gohugo.io/ rel="noopener noreferrer" target=_blank>Hugo</a> &
        <a href=https://github.com/adityatelange/hugo-PaperMod/ rel=noopener target=_blank>PaperMod</a></span></footer><a href=#top aria-label="go to top" title="Go to Top (Alt + G)" class=top-link id=top-link accesskey=g><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentcolor"><path d="M12 6H0l6-6z"/></svg></a><script>let menu=document.getElementById("menu");menu&&(menu.scrollLeft=localStorage.getItem("menu-scroll-position"),menu.onscroll=function(){localStorage.setItem("menu-scroll-position",menu.scrollLeft)}),document.querySelectorAll('a[href^="#"]').forEach(e=>{e.addEventListener("click",function(e){e.preventDefault();var t=this.getAttribute("href").substr(1);window.matchMedia("(prefers-reduced-motion: reduce)").matches?document.querySelector(`[id='${decodeURIComponent(t)}']`).scrollIntoView():document.querySelector(`[id='${decodeURIComponent(t)}']`).scrollIntoView({behavior:"smooth"}),t==="top"?history.replaceState(null,null," "):history.pushState(null,null,`#${t}`)})})</script><script>var mybutton=document.getElementById("top-link");window.onscroll=function(){document.body.scrollTop>800||document.documentElement.scrollTop>800?(mybutton.style.visibility="visible",mybutton.style.opacity="1"):(mybutton.style.visibility="hidden",mybutton.style.opacity="0")}</script><script>document.getElementById("theme-toggle").addEventListener("click",()=>{document.body.className.includes("dark")?(document.body.classList.remove("dark"),localStorage.setItem("pref-theme","light")):(document.body.classList.add("dark"),localStorage.setItem("pref-theme","dark"))})</script></body></html>